wordpress-mobile/AztecEditor-Android
# Introduce placeholders for android views

This PR is a draft for something I was working on recently. Our spannable approach doesn't allow showing of more complex views. For example it's impossible to create a video span that would play the video inline. Adding any type of a complex span is also very complicated. With this solution it is possible to add android views over a placeholder span. The Android view reacts to being deleted/moved around and works quite smoothly. The two placeholder drawers are currently just a POC to show it works. Only the code in the `aztec` repo should be +- final. Whatever is in the app module will be changed.

Questions raised from this PR are: 
- does the architecture look ok? I think I'll polish it a bit but a generally it should remain as it is
- does it work as expected? or is there something you'd polish?
- where should it live? it's currently in the `app` repo. I can either move it to a module (like the `glide-loader` module) or we can move it to our own repo and don't have it in Aztec. 

### Test
1. You'll notice when you start the app that there is already a test placeholder present
2. Try to delete it with backspace
3. Notice it works
4. You can add new placeholders by adding images or videos from your phone (I've replaced the functionality for testing)
    1. If you add a video, it should be shown and auto play (for testing purposes)
    2. If you add an image, it adds a test android layout which is clickable (shows a toast on click)
   

### Review
@AmandaRiu 

Make sure strings will be translated:
